### PR TITLE
linux: add tool to test release and beta builds on different distributions

### PR DESCRIPTION
**Pull request by remusao**
This script (which only **runs on Linux**) can be used to test Cliqz builds on a matrix of different distributions. The idea would be to allow automating some basic tests on the different operating systems that we support. This PR is only a first step towards that goal. The script is able to build a custom docker image and install Cliqz (`beta` or `release` versions), on any of the `<os>` supported (see below) as well as with a specified `<flavor>` (e.g. `rpm`, `deb`, `ppa`, `aur`); not all operating systems support all flavors. Here is the usage:

```
Usage: run.sh <channel> <os> <flavor> [<archive>]
  where:
    <channel> must be one of:
      - beta
      - release
      - local
    <os> must be one of:
      - ubuntu
      - ubuntu:16.04
      - ubuntu:18.04
      - ubuntu:19.04
      - ubuntu:19.10
      - debian
      - debian:10
      - debian:buster
      - opensuse
      - opensuse:15
      - opensuse:leap
      - opensuse:16
      - opensuse:tumbleweed
      - fedora
      - fedora:28
      - fedora:29
      - fedora:30
      - fedora:31
      - centos
      - centos:8
      - archlinux
    <flavor> must be one of:
      - ppa
      - deb
      - rpm
      - aur

Examples:
  ./run.sh release ubuntu ppa
  ./run.sh release debian:buster deb
  ./run.sh beta ubuntu:18.04 ppa
  ./run.sh beta opensuse:tumbleweed rpm
  ./run.sh local opensuse rpm ./foo/bar/cliqz.rpm
  ./run.sh local ubuntu deb ./foo/bar/cliqz.deb
```

### CODE REVIEW

**Comment by philipp-classen** [comment]
Nice tool. I had some stylistic comments, but overall it looks good!

File: docker/run.sh [hunk 1]
```diff
@@ -0,0 +1,470 @@
+#!/usr/bin/env sh
+
+set -e
+
```
[the remark this answers is not in the document]
Comment (remusao) [reply docker/run.sh]: Fixed in https://github.com/cliqz-oss/browser-f/pull/1444/commits/500e4eedb3b12b45abb80c6fcdb6ae9cd86ab145#diff-11d2ac8ae6a1aecc4757c3f28b1f1113R426.

File: docker/run.sh [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+usage() {
+  echo "Usage: run.sh <channel> <os> <flavor>"
+  echo "  where:"
+  echo "    <channel> must be one of:"
+  echo "      - beta"
+  echo "      - release"
+  echo "    <os> must be one of:"
+  echo "      - 16.04"
```
[the remark this answers is not in the document]
Comment (remusao) [reply docker/run.sh]: I unified the naming of `<os>`, thanks for the suggestion!

**Comment by remusao** [comment]
@alver-cliqz Good to go from my side. Let me know if you wanna discuss next steps there.

**Comment by remusao** [comment]
@philipp-classen I've updated the script with the ability to test a local `.deb` or `.rpm`, and also cleaned-up some aspects of the code. ` ./run.sh local ubuntu deb ./foo/bar/cliqz.deb`

**Pull request merged**